src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.DbRow.cs

```diff
@@ -51,19 +52,13 @@ internal readonly struct DbRow
 
             internal const int UnknownSize = -1;
 
-#if DEBUG
-            static unsafe DbRow()
-            {
-                Debug.Assert(sizeof(DbRow) == Size);
```

Comment:
Not sure why this assertion exists, but I reckon it's there to prevent regressions. cc @bartonjs

Reply:
Nvm, I see you added it to the main ctor.

Reply:
I just moved it to the ctor. I presume it would be nice to have `static_assert` in C# 🙂 if the size changes, the assert will hit anyway.